lua/dap/repl.lua

```diff
@@ -270,6 +270,9 @@ function M.append(line, lnum)
     if api.nvim_get_current_win() == repl.win and lnum == '$' then
       lnum = nil
     end
+    if api.nvim_buf_get_option(repl.buf, 'fileformat') ~= 'dos' then
+      line = line:gsub('\r\n', '\n')
```

Comment:
maybe `api.nvim_buf_get_option(repl.buf, 'fileformat')  '\n' or '\r'`?

Reply:
Do you mean to use a ternary expression for splitting?

Reply:
Both variants seem okay to me. Let me know how you decide and I'll merge

Reply:
I would leave the current change because I don't understand what @theHamsta means.

Reply:
I meant to convert to \n only on Unix, on Mac we need \r to have consistent line endings

Reply:
Makes sense! @mfussenegger, will you push the changes?